examples/widgets-gallery/gradle.properties

```diff
@@ -20,5 +20,5 @@ android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 kotlin.version=1.6.10
-compose.version=1.0.1-rc2
-agp.version=4.2.2
+compose.version=1.0.1
+agp.version=7.0.4
```

Comment:
Can we open project in IDEA this way?

Reply:
@olonho both import and android emulation worked fine with IntelliJ IDEA 2021.3.2 Ultimate